workplan.md

```diff
@@ -0,0 +1,42 @@
+# Solution.js v2 Workplan
+
+## 1 Create packages and Harness
+In order to keep things clean, we will create three new packages in the mono-repo. When this project is complete, the other packages will be removed.
+
+### [Core Package](./core-api)
+- the main orchestration functions
+- the solution specific interfaces
+- shared helper functions that are specific to Solutions
+  - more generic helpers should be PR'd to rest-js
+
+### [Processor Package](./processor-api)
+- contain the item type specific processors
+- depends on the `core` package
+
+### Harness App
+- in the /harness folder, create a react app that will act as a debugging harness during development
+- ensures ESM output is viable and optimized
+- use create-react-app and react-router and calcite-components
```

Comment:
new components should be considered for Stencil (hub-components) so they can be re-used in Online and other products